client/src/components/responseMatching/ActiveMatching.vue

```diff
@@ -22,9 +22,6 @@
       v-tooltip.left.ds900="$t('tooltip.noMatch', { response: selectedResponse })"
       class="btn btn-primary w-50 fs-4 d-flex justify-content-center align-items-center"
       @click="noMatch">
-      <div class="d-flex justify-content-center align-items-center w-75">
```

Comment:
would like to keep the image inside the button positioned and sized as close to what it looks like currently. But hopefully following the expected html5 idioms: https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button